# HG changeset patch
# User Rik <rik@octave.org>
# Date 1411789885 25200
#      Fri Sep 26 20:51:25 2014 -0700
# Node ID 7671033039748c171177ab265d3a0757afa6d041
# Parent  8553b88e59094b4a0a7c2fa85450b26ab3737716
doc: Re-order seealso references in rot90, rotdim.

* rot90.m, rotdim.m: Put fliplr ahead of flipud in seealso references
(alphabetized).

diff --git a/scripts/general/rot90.m b/scripts/general/rot90.m
--- a/scripts/general/rot90.m
+++ b/scripts/general/rot90.m
@@ -46,17 +46,17 @@
 ## rot90 ([1, 2; 3, 4], 7)
 ## @end group
 ## @end example
 ##
 ## The rotation is always performed on the plane of the first two dimensions,
 ## i.e., rows and columns.  To perform a rotation on any other plane, use
 ## @code{rotdim}.
 ##
-## @seealso{rotdim, flipud, fliplr, flip}
+## @seealso{rotdim, fliplr, flipud, flip}
 ## @end deftypefn
 
 ## Author: jwe
 
 function B = rot90 (A, k = 1)
 
   if (nargin < 1 || nargin > 2)
     print_usage ();
diff --git a/scripts/general/rotdim.m b/scripts/general/rotdim.m
--- a/scripts/general/rotdim.m
+++ b/scripts/general/rotdim.m
@@ -46,17 +46,17 @@
 ##
 ## @example
 ## @group
 ## rotdim ([1, 2; 3, 4], -1, [1, 2])
 ## rotdim ([1, 2; 3, 4], 3, [1, 2])
 ## rotdim ([1, 2; 3, 4], 7, [1, 2])
 ## @end group
 ## @end example
-## @seealso{rot90, flipud, fliplr, flip}
+## @seealso{rot90, fliplr, flipud, flip}
 ## @end deftypefn
 
 function y = rotdim (x, n, plane)
 
   if (nargin < 1 || nargin > 3)
     print_usage ();
   endif
 
